index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Music History!</title>
+	<link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+<body>
+	<nav>
+		<ul>
+			<li>View Music</li>
+			<li><a href="#">Add Music</a></li>
+			<li><a href="#">Profile</a></li>
+		</ul>
+	</nav>
+	<div class="input">
+			<br>Song Length:<br>
+			<input type="range" id="songlength" min="" max="" step="">
+
```

Comment:
Looking really great here as well. Eventually you'll need all your values filled out.